### ISSUE TITLE
obtaining mAP 0.75 score for voc format dataset training

### ISSUE THREAD

**Issue by buyukkanber** [Megvii-BaseDetection/YOLOX#1617]
Hi!

How can I get the mAP 0.75 score with eval.py on the test dataset from the best_ckpt.pth weight, when I train my custom dataset which has voc format? 

Thanks

**Comment by buyukkanber** [comment]
Much obliged, 

But once I change the [overlap threshold value](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/evaluators/voc_eval.py#L76) to 0.75 and run the [eval.py](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/tools/eval.py) as instructed in the [README.md](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/README.md) file 
`python -m yolox.tools.eval -n  yolox-l -c YOLOX_outputs/yolox_voc_l/best_ckpt.pth -b 1 -d 1 --conf 0.001 --fp16 --fuse`,
I still get the same output values that equals to mAP_0.50 and mAP_5095 scores respectively.

**Comment by FateScript** [comment]
Did you modify code [here](https://github.com/Megvii-BaseDetection/YOLOX/blob/4f8f1d79c8b8e530495b5f183280bab99869e845/yolox/data/datasets/voc.py#L306)?

**Comment by buyukkanber** [comment]
No, I haven't changed anything in the [voc.py](https://github.com/Megvii-BaseDetection/YOLOX/blob/4f8f1d79c8b8e530495b5f183280bab99869e845/yolox/data/datasets/voc.py).

**Comment by FateScript** [comment]
@buyukkanber Plz modify the code to fit your need.
